### PR TITLE
Create a TMDB mapping

### PR DESCRIPTION
**Pull request by Soitora**
Mapped **TMDB** using the **TVDB** file along with [my helper](https://github.com/Soitora/TMDB-PAS-Assistant) program.

Some things are not named in the right order, as **TMDB** and **TVDB** has different names, and I went in straight order due to the sheer amount of series.

I checked 90+% of them manually, so it should close to a perfect map

No clue how the **Sort** or **Validate** file treats this, I used the URL link styled discussed here: RickDB/PlexAniSync#193

### CODE REVIEW

File: series-tmdb.en.yaml [hunk 1]
```diff
@@ -0,0 +1,2683 @@
...
+    # https://www.themoviedb.org/tv/67674
+    # https://www.imdb.com/title/tt6027642/
+    synonyms:
+      - "Danganronpa 3 The End Of Kibougamine Gakuen Zetsubou Hen"
+      - "Danganronpa 3 Zetsubou Hen"
+    seasons:
+      - season: 1
+        anilist-id: 21825
+
+  # PlexAniSync does not support mapping of Specials (Gaiden, 4 episodes)
```
Comment (reconman) [inline series-tmdb.en.yaml:497]: Comment needs to be under title so sort.py doesn't associate it with Danganronpa 3.

File: series-tmdb.en.yaml [hunk 2]
```diff
@@ -0,0 +1,2683 @@
...
+      - "Demon King Mismatched School ~The Founder of the Strongest Demon King in History, He Arrives at School as Descendants After Reincarnating~"
+      - "Maou Gakuin no Futekigousha ~Shijo Saikyou no Maou no Shiso, Tenseishite Shison-tachi no Gakkou e Kayou~"
+      - "Maou Gakuin no Futekigousha: Shijou Saikyou no Maou no Shiso, Tensei Shite Shison-tachi no Gakkou e Kayou"
+      - "Maō Gakuin no Futekigōsha ~Shijo Saikyō no Maō no Shiso, Tenseishite Shison-tachi no Gakkō e Kayō~"
+      - "Maō Gakuin no Futekigōsha: Shijō Saikyō no Maō no Shiso, Tensei shite Shison-tachi no Gakkō e"
+      - "The Misfit of Demon King Academy: History's Strongest Demon King Reincarnates and Goes to School with His Descendants"
+      - "The Unqualified Student at the Demon Lord Academy ~ The Strongest Demon Lord in History, the Founder, Reincarnates and Attends a School With His Descendants ~"
+      - "The Unqualified Student of Demon Lord Academy ~The Strongest Demon Lord in History, the Founder, Attends the School of his Descendants After Reincarnation~"
+      - "魔王学院"
+      - "🈟魔王学院の不適合者"
```
Comment (reconman) [inline series-tmdb.en.yaml:2280]: sort.py turns `🈟` into `\U0001F21F`. Not sure if that causes problems or not.

**Review by reconman — CHANGES_REQUESTED** [review]
You should run
```
pip install requirements.txt
python sort.py
```
, fix any issues caused by that and commit the result.

File: series-tmdb.en.yaml (hunk 2 above)
Comment (Soitora) [reply series-tmdb.en.yaml:2280]: If anything, it detected an odd symbol that for some reason is part of the Synonyms, which is good, removed.

**Comment by Soitora** [comment]
> You should run [the sorter], fix any issues caused by that and commit the result.

Done

**Review by reconman — APPROVED** [review]

**Pull request merged**